### DISCUSSION TITLE
Conditional block

### DISCUSSION THREAD

**Discussion by shritasharma** [postmanlabs/postman-flows#313] [Q&A] [answered]
Hi,
New to the flow in Postman. I am trying to add expression in a conditional block and I am getting an error "TypeError: Expected symbol == (string) to be of type [] " .
The only thing I am trying to do is to compare the value of the attribute in the response of the previous block with "null"

Has anyone faced this problem before?
Thanks,
Shrita.
![image](https://user-images.githubusercontent.com/28731302/167515146-f9f4bbf1-e56c-42d4-b86a-b3ddfeafd705.png)

**Comment by DmytroKryvovychev** [comment, marked as answer]
Hi, @shritasharma 
i've had the same problem.
Try this solution
![image](https://user-images.githubusercontent.com/67005033/167809115-987f8c39-eaed-4bba-be83-2008551bbea1.png)

**Reply by shritasharma** [reply]
Thanks so much it did the trick!
